docs/codeql/support/reusables/versions-compilers.rst

```diff
@@ -17,11 +17,11 @@
 
    .NET 5","``.sln``, ``.csproj``, ``.cs``, ``.cshtml``, ``.xaml``"
    Go (aka Golang), "Go up to 1.16", "Go 1.11 or more recent", ``.go``
-   Java,"Java 7 to 15 [3]_","javac (OpenJDK and Oracle JDK),
+   Java,"Java 7 to 15 [3]_, Java 16 (partially)","javac (OpenJDK and Oracle JDK),
```

Comment:
```suggestion
   Java,"Java 7 to 16 [3]_","javac (OpenJDK and Oracle JDK),
```

Also update the footnote [3] further below to:
```
[3] Builds that execute on Java 7 to 16 can be analyzed. The analysis understands Java 15 standard language features.
```

Reply:
Thanks for committing the suggested change. Please also update the footnote as above.

Reply:
It should be fine now.